README.md

```diff
@@ -70,25 +78,27 @@ _There are more potential improvements to be made._
 _Still, those are yet to be defined and properly implemented within a future parachain project - since it's hard to predict what exactly the parachain might need or not need._
 _The current solution is general and parachain-agnostic._
 _The MoveVM is taken from the Move language repository - any possible safety issues are inherited from that repo._
-_Therefore, the substrate-move fork should get all upstream changes from that repository._
-_The first parachain adapters should do additional testing to ensure the solution is robust enough._
+_Therefore, the Substrate MoveVM fork ([substrate-move][substrate-move]) should get all upstream changes from that repository._
+_The first parachain adapters should do additional testing to ensure the robustness of the solution._
 
 
 ## Open / Planned Points
 
-- Refine gas tuning and gas-weight logic
+- Completely define/refine gas tuning and gas-weight logic
 - Update to the latest Polkadot-SDK version in template node / pallet-move
 - Add more pallet tests
 - Develop full benchmarks
-- Multi-signature support
+- Multi-signature support!
 - Protection mechanism for Move address 0x1
-- Add documentation about Substrate-stdlib
-- Restructure GitHub repositories of Substrate-stdlib and Move-stdlib
-
+- Move `Move-stdlib` repository location
+- Add an extrinsic for updating `Move-stdlib` with special root account
+- Improve RPC commands (and `smove` commands)
```

Comment:
That point is unclear to me - so what's wrong with the current RPC and smove commands, someone may ask.

Reply:
Done